### DISCUSSION TITLE
✅ Telnet

### DISCUSSION THREAD

**Discussion by probiotisch** [isontheline/pro.webssh.net#228] [Ideas]
Please add Telnet 👍

**Comment by isontheline** [comment]
Hello @probiotisch 👋

Thank you for your suggestion!

**Comment by BruceAC** [comment]
Yes, please add telnet.  When I am trying to troubleshoot some older devices, I'd rather use WEBSSH on my Mac than telnetting from Linux.

**Reply by isontheline** [reply]
Thank you a lot for your feedback @BruceAC 

It will help drive the roadmap

**Comment by isontheline** [comment]
Now available inside the latest WebSSH release
